<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>game_select</class>
 <widget class="QWidget" name="game_select">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>130</y>
     <width>131</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="no_die_mode">
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;华文楷体&quot;;</string>
      </property>
      <property name="text">
       <string>无限模式</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="limited_time_mode">
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;华文楷体&quot;;</string>
      </property>
      <property name="text">
       <string>限时模式</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ultimate_mode">
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;华文楷体&quot;;</string>
      </property>
      <property name="text">
       <string>多蛇运动</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
